packages/react-components/src/components/styles/HorizontalGallery.styles.ts

```diff
@@ -15,6 +15,7 @@ export const leftRightButtonStyles = (theme: Theme): IStyle => {
   return {
     background: 'none',
     padding: 0,
+    zIndex: 1,
```

Comment:
css properties are very arbitrary and hard to understand why they exist (and thus changing css often causes regressions elsewhere). Add a comment here to say why zIndex is set here (i.e. that the buttons must exist in a higher z band than <insert other element that is causing this bug>)

Reply:
+1

Actually, I like to define constants side-by-side for all elements that have relative z-index.

e.g.

```ts
// Relative zIndex values to stack contents of the horizontal gallery 
HORIZONTAL_GALLERY_CONTAINER_ZINDEX = 0
HORIZONTAL_GALLERY_NEXT_BUTTON_ZINDEX = 1
HORIZONTAL_GALLERY_PREV_BUTTON_ZINDEX = 1
```

That way, it's obvious what the stacking is like.

Reply:
Done, added comment and also used constant for adding higher zIndex for left/right buttons.

Reply:
Sorry to join the discussion late but would a better alternative be setting the CSS property 'pointerEvents' of the layerHost of VideoGallery to be 'none' so that it does not block any other buttons we may add to VideoGallery in the future. It might get tedious to keep applying zIndex properties in VideoGallery elements. What do you guys think?

Reply:
^if that solves it that would be a much cleaner fix, especially if we introduce video gallery tiles that are interactive